src/logger.ts

```diff
@@ -0,0 +1,12 @@
+import * as winston from 'winston'
+
+const logger = winston.createLogger({
```

Comment:
This is not the best approach. It's a library that should not do any default configuration of the logger.